api/src/main/java/ca/bc/gov/educ/api/edx/struct/v1/EdxUserDistrictActivationRelinkSagaData.java

```diff
@@ -5,12 +5,11 @@
 import lombok.EqualsAndHashCode;
 
 import jakarta.validation.constraints.NotNull;
-import java.io.Serializable;
 
 @EqualsAndHashCode(callSuper = true)
 @Data
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class EdxUserDistrictActivationRelinkSagaData extends EdxUserDistrictActivationInviteSagaData implements Serializable {
```

Comment:
Is this something we should be removing? I thought we needed it on our structs

Reply:
Yep!  Serializable is already inherited by `EdxUserDistrictActivationInviteSagaData`, and so, is redundant here.